doc/releases/changelog-0.41.0.md

```diff
@@ -540,24 +564,65 @@ capture enabled by adding :func:`qml.capture.enable() <pennylane.capture.enable>
   Also added ``qml.pauli.trace_inner_product`` that can handle batches of dense matrices.
   [(#6811)](https://github.com/PennyLaneAI/pennylane/pull/6811)
 
-* Added class ``qml.FromBloq`` that takes Qualtran bloqs and translates them into equivalent PennyLane operators. For example, we can now import Bloqs and use them in a way similar to how we use PennyLane templates:
+* ``qml.structure_constants`` now accepts and outputs matrix inputs using the ``matrix`` keyword.
+  [(#6861)](https://github.com/PennyLaneAI/pennylane/pull/6861)
+
+<h4>Qualtran Integration 🔗</h4>
+
+* It's now easy to use [Qualtran](https://qualtran.readthedocs.io/en/latest/) bloqs in PennyLane. The new ``qml.FromBloq`` class translates [Qualtran bloqs](https://qualtran.readthedocs.io/en/latest/bloqs/index.html#bloqs-library) into equivalent PennyLane operators [(#7148)](https://github.com/PennyLaneAI/pennylane/pull/7148).
+  
+  `qml.FromBloq` requires two inputs:
+  * bloq: an initialized Qualtran Bloq
+  * wires: the wires the operator acts on
+  
+  The following example applies a PennyLane Operator and Qualtran Bloq in the same circuit:
+
   ```python
   >>> from qualtran.bloqs.basic_gates import CNOT
   
-  >>> dev = qml.device("default.qubit") # Execute on device
+  >>> dev = qml.device("default.qubit")
   >>> @qml.qnode(dev)
   ... def circuit():
+  ...    qml.X(wires=0)
   ...    qml.FromBloq(CNOT(), wires=[0, 1])
   ...    return qml.state()
   >>> circuit()
-  array([1.+0.j, 0.+0.j, 0.+0.j, 0.+0.j])
+  array([0.+0.j, 0.+0.j, 0.+0.j, 1.+0.j])
   ```
-  [(#7148)](https://github.com/PennyLaneAI/pennylane/pull/7148)
 
-* ``qml.structure_constants`` now accepts and outputs matrix inputs using the ``matrix`` keyword.
-  [(#6861)](https://github.com/PennyLaneAI/pennylane/pull/6861)
+* The new ``qml.bloq_registers`` helps to determine the required wires for more complicated ``Bloqs``.
```

Comment:
```suggestion
* A new function called :func:`qml.bloq_registers <pennylane.bloq_registers>` is available to help determine 
  the required wires for more complicated Qualtran ``Bloqs``.
```

Reply:
Also, can we give an example of a general condition for which I'd need to use `bloq_registers`?